Python/Product/Analysis/Analyzer/ExpressionEvaluator.cs

```diff
@@ -446,6 +446,10 @@ internal class ExpressionEvaluator {
         }
 
         internal void AssignTo(Node assignStmt, Expression left, IAnalysisSet values) {
+            if (left is ExpressionWithAnnotation) {
+                left = ((ExpressionWithAnnotation)left).Expression;
+            }
+
             if (left is NameExpression) {
                 var l = (NameExpression)left;
```

Comment:
We actually never evaluate the LHS of an assignment statement... I wonder how much will break if I enable that...